poetry/core/masonry/builders/wheel.py

```diff
@@ -181,7 +181,7 @@ def _write_metadata(self, wheel):
                 relative_path = "%s/%s" % (self.dist_info, path.relative_to(self._path))
                 self._add_file(wheel, path, relative_path)
             else:
-                logger.debug("Skipping: %s", path.as_posix())
+                logger.debug("Skipping: %s" % path.as_posix())
```

Comment:
For logging it sould be `,`, this should defer evaluation to only when it is required.

Reply:
Hm, I found this code accidentally, because my console printed out `Skipping: %s`. I've  replaced this now with `.fortmat(...)`